src/content/docs/en/pages/main-menu/reference/observe/real-time-events/real-time-events.mdx

```diff
@@ -41,261 +41,271 @@ You can use Real-Time Events to:
-| $typereload | Type of navigation for the reload operation: when the page was reloaded. Example: `1` |
-| $typereserved | Any type of navigation not defined by other specific variables (**typeNavigate** and **typeReload**). Example: `1` |
-| $userAgent | End user’s application, operating system, vendor, and/or version. Value of the `User-Agent` header. Example: `Mozilla/5.0 (Windows NT 10.0; Win64; x64)` |
-| $version | The Azion Log version used. Example: `v5` |
+| -------- | ----------- |
+| Bytes Sent | Number of bytes sent to a client. This field is the result of a sum. Example: `191` |
+| Configuration ID | Unique Azion configuration identifier set on virtual host configuration file. Example: `1595368520` |
+| Host | Host information sent on the request line. Stores: host name from the request line, or host name from the `Host` request header field, or the server name matching a request. Example: `g1sdetynmxe0ao.map.azionedge.net` |
+| HTTP Referer | Address of the page the user made the request from. Example: `https://example.com` |
+| HTTP User Agent | End user’s application, operating system, vendor, and/or version. Value of the `User-Agent` header. Example: `Mozilla/5.0 (Windows NT 10.0; Win64; x64)` |
+| Reference Error | Reference ID of the request. Generated when the status code is bigger than 400. Example: `#AECFE66100000000C947B9B3B3BFBE46FFFFFFFF9401` |
+| Remote Addr | IP address of the origin that generated the request. Example: `127.0.0.1` |
+| Remote Port | Port of the origin that generated the request. Example: `8080` |
+| Request Method | HTTP request method. Example: `GET` or `POST` |
+| Request Time | Request processing time elapsed since the first bytes were read from the client with resolution in milliseconds. This field is the result of a sum. Example: `1.19` |
+| Request Uri | URI of the request made by the end user, without the host and protocol information and with arguments. Example: `/v1?v=bo%20dim` |
+| Scheme | Request scheme. Example: `HTTP` or `HTTPS` |
+| Solution | Identifier of your edge application. Example: `1321` |
+| SSL Cipher | Cipher string used to establish TLS connection. Example: `TLS_AES_256_GCM_SHA384` |
+| SSL Protocol | Protocol for an established TLS connection. Example: `TLS v1.2` |
+| SSL Session Reused | Returns `r` if an SSL session was reused or `.` if it wasn't. |
+| Status | HTTP status code of the request. Example: `200` |
+| TCP Info RTT | Round-Trip Time (RTT) measured by the edge for the user. Available on systems that support the TCP_INFO socket option. Example: `72052` |
+| Upstream Cache Status | Status of the local edge cache. Can be: `MISS`, `BYPASS`, `EXPIRED`, `STALE`, `UPDATING`, `REVALIDATED`, `HIT`, or `-` |
+| Upstream Response Time | Time it takes for the edge to receive a default response from the origin in milliseconds, including headers and body. This field is the result of a sum. Example: `0.876`. **In case of cache, the response is `-`**. |
```

Comment:
```suggestion
| Upstream Response Time | Time, in milliseconds, for the edge to receive a default response from the origin, including headers and body. This field is the result of a sum. Example: `0.876`. **In case of cache, the response is `-`**. |
```